node-operators/validators/onboarding/opt-in-to-tanssi.md

```diff
@@ -0,0 +1,282 @@
+
+To follow this guide, you must interact with smart contracts, one from Symbiotic's core protocol and the other from Tanssi's integration with Symbiotic.
+
+--8<-- 'text/node-operators/validators/onboarding/run-a-validator/prerequisites.md'
+
+## Opt In to Tanssi-Enabled Vaults {: #opt-in-tanssi-vaults }
+
+Before enabling your operator to be an active Tanssi network validator, you must opt in to at least one of the Tanssi-enabled vaults. The following sections describe several ways to opt into the vaults.
+
+### Opt In Using the Symbiotic CLI {: #opt-in-tanssi-vaults-with-cli }
+
+If you have correctly installed the [Symbiotic CLI](#setting-up-the-cli) and you want to sign the transaction using a Ledger device, then run the following command, replacing the `INSERT_VAULT_ADDRESS` with the specific one you want to join and `INSERT_OPERATOR_ADDRESS` with your account:
+
+=== "MainNet"
+
+    ```bash
+    python3 symb.py opt-in-vault INSERT_VAULT_ADDRESS --ledger --ledger-account INSERT_OPERATOR_ADDRESS
+    ```
+
+=== "TestNet (Sepolia)"
+
+    ```bash
+    python3 symb.py opt-in-vault {{ networks.symbiotic.contracts.sepolia.vault }} --ledger --ledger-account INSERT_OPERATOR_ADDRESS --chain sepolia
+    ```
+
+If you want to sign the transaction directly using the operator's account private key, then run the following command, replacing the `INSERT_PRIVATE_KEY` parameter:
+
+=== "MainNet"
+
+    ```bash
+    python3 symb.py opt-in-vault INSERT_VAULT_ADDRESS --private-key INSERT_PRIVATE_KEY
+    ```
+
+=== "TestNet (Sepolia)"
+
+    ```bash
+    python3 symb.py opt-in-vault {{ networks.symbiotic.contracts.sepolia.vault }} --private-key INSERT_PRIVATE_KEY --chain sepolia
+    ```
+
+!!! warning
+    Note that this method requires you to expose your private key; therefore, it is not recommended.
+
+### Opt In Using Etherscan {: #opt-in-tanssi-vaults-with-etherscan }
+
+You can interact with Symbiotic's smart contracts using Etherscan, and sign the transaction using a browser wallet ([MetaMask](/builders/toolkit/ethereum-api/wallets/metamask/#install-the-metamask-extension){target=\_blank}, for example). 
+
+To open the contract's page, open the link:
+
+=== "MainNet"
+
+    [Contract address: {{ networks.symbiotic.contracts.mainnet.opt_in_vault }}](https://etherscan.io/address/{{ networks.symbiotic.contracts.mainnet.opt_in_vault }}#writeContract){target=\_blank}
+
+=== "TestNet (Sepolia)"
+
+    [Contract address: {{ networks.symbiotic.contracts.sepolia.opt_in_vault }}](https://sepolia.etherscan.io/address/{{ networks.symbiotic.contracts.sepolia.opt_in_vault }}#writeContract){target=\_blank}
+
+Click on **Connect to Web3**, and select your preferred wallet (e.g. MetaMask):
+
+![Connect to Web3 step](/images/node-operators/validators/onboarding/opt-in-to-tanssi/opt-in-to-tanssi-1.webp)
+
+!!! note
+    You can configure MetaMask to use a cold wallet.
+
+Once connected:
+
+1. Expand the `optin` function
+2. Insert the `VAULT_ADDRESS`
```

Comment:
If UI elements, please put em in bold too! Please check rest of this guide too for these instances